### PR TITLE
Clem

### PR DESCRIPTION
**Pull request by clemcode**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -7,5 +7,11 @@ module.exports = {
       "eol-last": 0,
       "no-param-reassign": 0,
       "no-console": 0,
+      "class-methods-use-this": 0,
     },
+    "globals": {
+      "document": true,
+      "localStorage": true,
+      "window": true
+    }
```
Comment (DavidBruant) [inline .eslintrc.js]: probably better solved with an env http://eslint.org/docs/user-guide/configuring#specifying-environments
`env: browser`

File: src/components/pages/Login.vue [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="login">
+    <p>Pour vous connecter, merci de vous créer un compte sur Github et de rentrer votre Access Token</p>
+    <form action="GET" v-on:submit.prevent="verifyUser()">
+      <label for="token">Github Access Token</label>
+      <input type="text" v-model="token">
+      <input type="submit" value="Yo">
```
Comment (DavidBruant) [inline src/components/pages/Login.vue]: button, not input

Comment (DavidBruant) [reply src/components/pages/Login.vue]: What does this component look like if there is already a token in the localStorage?

File: src/loginStore.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      token,
+      name,
+    };
+  }
+
+  setLogin(token, name) {
+    this.token = token;
+    this.name = name;
+    localStorage.setItem('token', token);
+    localStorage.setItem('name', name);
```
Comment (DavidBruant) [inline src/loginStore.js:14]: Why save the name?

**Review by DavidBruant — CHANGES_REQUESTED** [review]

File: src/loginStore.js (hunk 3 above)
Comment (clemcode) [reply src/loginStore.js:14]: Maybe for displaying user name on header when logged in, but not mandatory

**Pull request merged**